test-scripts/all.test

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+apertium-quality
+
+echo 'WER uzb-kaa:'
+perl ../apertium-eval-translator/apertium-eval-translator-line.pl -test texts/story-uzb-kaa.txt -ref texts/story-kaa.txt > texts/uzb-kaa-wer.txt
+grep '(WER)' texts/uzb-kaa-wer.txt
+
+echo 'WER kaa-uzb:'
+perl ../apertium-eval-translator/apertium-eval-translator-line.pl -test texts/story-kaa-uzb.txt -ref texts/story-uzb.txt > texts/kaa-uzb-wer.txt
+grep '(WER)' texts/kaa-uzb-wer.txt
+
```

Comment:
These scripts can just go in `tests/`, you don't need a separate directory for them.  Call this one `wer.sh` or similar.